src/components/capture-area/RegexTextField/RegexTextField.test.ts

```diff
@@ -0,0 +1,105 @@
+import { describe, it, expect } from 'vitest';
+import { ref } from 'vue';
+import { shallowMount, mount } from '@vue/test-utils';
+import RegexTextField from '@/components/capture-area/RegexTextField/RegexTextField.vue';
+import { Vigad } from '@/proc/Vigad';
+import {
+    defaultValueRegexSettings,
+    defaultAfterConstraintsSettings,
+    defaultBeforeConstraintsSettings,
+} from '@/components/capture-area/CaptureAreaSearchValue/ResetSettings';
+
+describe('RegexTextField', () => {
+    /**
+     * Vigad instance
+     */
+    const vigad = ref(Vigad.getInstance());
+
+    /**
+     * Add capture area
+     */
+    vigad.value.addCaptureArea(100, 100, 0, 0);
+
+    /**
+     * Capture areas
+     */
+    const captureAreas = ref(vigad.value.getAllCaptureAreas());
+
+    /**
+     * Value regex
+     */
+    const valueRegex = vigad.value
+        .getCaptureArea(captureAreas.value[0].getId())
+        .getRegexGroups()[0]
+        .getValueRegex();
+
+    /**
+     * Before constraint
+     */
+    const beforeConstraint = vigad.value
+        .getCaptureArea(captureAreas.value[0].getId())
+        .getRegexGroups()[0]
+        .getConstraintRegex()[0];
+
+    /**
+     * After constraint
+     */
+    const afterConstraint = vigad.value
+        .getCaptureArea(captureAreas.value[0].getId())
+        .getRegexGroups()[0]
+        .getConstraintRegex()[1];
+
+    it('render with value regex', async () => {
+        const props = {
+            label: 'Search value',
+            placeholder: 'Enter search value',
+            prependIcon: 'mdi-table-column',
+            captureAreaId: captureAreas.value[0].getId(),
+            regex: valueRegex,
+            resetOptions: defaultValueRegexSettings,
+        };
+
+        const wrapper = mount(RegexTextField as any, {
+            props: props,
+        });
+
+        expect(wrapper.html());
```

Comment:
this just checks whether something exists inside the wrapper right?

Reply:
yes